README.md

```diff
@@ -1,68 +1,35 @@
 # Ozone Distribution For Cambodia 🇰🇭
 
-Ozone - All in one system for healthcare facilities: https://www.ozone-his.com/
+A Maven project to download and package in one place all the software artefacts that make the Ozone Distribution for Cambodia.
 
------
+## Summary
 
-Maven project for Ozone distribution for Cambodia. Downloads and brings in one place all artifacts needed by the distribution
+The Ozone Distribution For Cambodia or **Ozone Kh** is a distribution of [Ozone HIS](https://www.ozone-his.com) specially configured for the MoH of Cambodia. In its original form it is an integrated system between an NCD-capable EMR system and a reporting & analytics platform for NCD population reporting.
 
+## Demo Environments
 
-## Implementer instructions
+| Server   | URL                                        | Credentials      |
+|----------|--------------------------------------------|------------------|
+| Dev      | https://oz-kh-dev.mekomsolutions.net       | admin / Admin123 |
+| Training | https://oz-kh-training.mekomsolutions.nett | admin / Admin123 |
```

Comment:
`tt` -> `https://oz-kh-training.mekomsolutions.net`

But I would point to the full URL for now (since we have an issue with redirection to `spahome`.
`https://oz-kh-dev.mekomsolutions.net/openmrs/spa/login#`
`https://oz-kh-training.mekomsolutions.net/openmrs/spa/login#`